relays/utils/Cargo.toml

```diff
@@ -10,10 +10,13 @@ ansi_term = "0.12"
 async-std = "1.6.5"
 async-trait = "0.1.40"
 backoff = "0.2"
+chttp = "0.5"
```

Comment:
Any specific reason for a new http client? Or in other words, any specfic reason not to use `hyper` which is a dependency of Substrate anyways?

Reply:
Afair `hyper` requires tokio runtime, which we don't use + different `hyper` versions are requiring different tokio runtimes, making it harder to maintain. The `chttp` is runtime-agnostic => easier to use

Reply:
> hyper which is a dependency of Substrate anyways?

We're not in the substrate binary ;)

Reply:
Good point, wasn't sure about `tokio`. Don't like it to much anyways :-)